### PR TITLE
[WIP][DO NOT MERGE]make retry interval longer and log the user direct

### PR DESCRIPTION
**Pull request by IBMCaoBiao**
because there are still errors of unable to link disks.
So submit this PR to try to make the retry interval longer. Also log the user directory for debug.
Signed-off-by: SharpRazor <bjcb@cn.ibm.com>

### CODE REVIEW

File: smtLayer/vmUtils.py [hunk 1]
```diff
@@ -330,6 +330,10 @@ def installFS(rh, vaddr, mode, fileSystem, diskType):
                                    e.returncode, e.output))
                     time.sleep(sleep_secs)
                 else:
+                    res = invokeSMCLI(rh, "Image_Query_DM", ['-T', rh.userid])
+                    user_direct = res['response']
+                    rh.printSysLog("Failed to link %s, its user directory "
+                                   "is:%s." % (rh.userid, user_direct))
```
Comment (bjhuangr) [inline smtLayer/vmUtils.py:336]: if new exception raised from new added code, will overwrite origin exception

Comment (SharpRazor) [reply smtLayer/vmUtils.py:336]: you mean I should remove the 'raise' sentence?

**Comment by SharpRazor** [comment]
> Be aware, this change will increase zcc smapi calls

I think this just for debug, if we found the issue, we can remove this.

**Comment by jichenjc** [comment]
less likely for more SMAPI calls as this is unexpected case..

**Comment by SharpRazor** [comment]
@jichenjc considering this PR is more likely for debug, I can edit codes in infra team's server.
And this PR just for review and do not merge. is this OK?

**Comment by bjhuangr** [comment]
close the inactive PR

**Pull request closed without merging**